### DISCUSSION TITLE
Zoom SDK Help

### DISCUSSION THREAD

**Discussion by xor83** [remix-run/remix#5333] [Q&A] [answered]
I am trying to integrate Zoom SDK mentioned on this page:
https://www.npmjs.com/package/@zoomus/websdk#usage-client-view

SDK import is giving me error
import { ZoomMtg } from '@zoomus/websdk'

Error:
ReferenceError: self is not defined
at node_modules/@zoomus/websdk/dist/zoomus-websdk.umd.min.js (\test-remix\build\index.js:2275:8)

Any help?

**Comment by machour** [comment, marked as answer]
You probably need to wrap it in [`<ClientOnly />`](https://github.com/sergiodxa/remix-utils#clientonly)

**Reply by xor83** [reply]
Thanks machour for the reply. `<ClientOnly />` Worked for other package I am having same issue but for zoom I added .client to filename and it worked.

Thank you.
